src/Home.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
```

Comment:
I think it's easier if you have the name of the file the same as the name of the component.
What do you think @manu?

Reply:
ya @manu WDYT?
<img width="591" alt="image" src="https://github.com/BinaHovav/MyMeals-Frontend/assets/58728824/644bbfe7-e452-497b-8825-77213beac39b">

Reply:
I agree @shulajz

Reply:
fixed that

Reply:
u didn't @BinaHovav - the name of the file is still `Home` and the name of the component is `HomePage`

Reply:
true

Reply:
ngltt fixed that

Reply:
(not gonna lie this time)